pkg/diff/diff_test.go

```diff
@@ -0,0 +1,318 @@
+package diff_test
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/MaineK00n/vuls-data-update/pkg/diff"
+)
+
+func TestDiff(t *testing.T) {
+	if os.Getenv("GITHUB_ACTIONS") != "true" {
+		t.Skip("only on GitHub Actions")
+	}
```

Comment:
Why limit unit tests to only run on Github Actions?

Reply:
Hmm,,, two not very strong reasons.
1. they require internet access, so it fails in case of no internet access.
2. they takes a little long time (compared to other existing unit tests)

Reply:
Change unit test to use local git daemon and removed this. Thanks!